frontend/src/app/page.tsx

```
@@ -168,14 +168,27 @@ export default function Home() {
             {data.upcomingEvents.map((event: Event, index: number) => (
               <div key={`card-${event.name}`} className="overflow-hidden">
                 <div className="rounded-lg bg-gray-200 p-4 dark:bg-gray-700">
-                  <div className="mb-2 flex items-center justify-between">
+                  <div className="mb-2 flex items-center justify-between gap-2">
                     <button
+                      className="min-w-0 flex-1 cursor-pointer text-left text-lg font-semibold text-blue-400 hover:underline"
```

Biome (2.1.2)

[error] This JSX property is assigned multiple times.

This attribute is assigned again here.


(lint/suspicious/noDuplicateJsxProps)